File: Sorting/bubble sort/php/bubblesort.php
<?php
$arr = array(2,5,6,1,9);
function bubble_sort($arr) {
    $size = count($arr)-1;
    for ($i = 0; $i < $size; $i++) {
        for ($j = 0; $j < $size - $i; $j++) {
            $k = $j + 1;
            if ($arr[$k] < $arr[$j]) {
                // Swap elements at indices: $j, $k
                list($arr[$j], $arr[$k]) = array($arr[$k], $arr[$j]);
            }
        }
    }
    for ($i = 0; $i < $size; $i++)
        echo $arr[$i];
}
?>

File: Sorting/bubble sort/python/BubbleSort.py
def bubbleSort(arr):
    swap = True
    while swap:
        swap = False
        for i in range(len(arr) - 1):
            if arr[i] > arr[i + 1]:
                x = arr[i]
                arr[i] = arr[i + 1]
                arr[i + 1] = x
                swap = True


if __name__ == "__main__":
    arr = [10, -43, 21, 0, 4, 67, 5, 3, 10]
    bubbleSort(arr)
    print(arr)
